### PR TITLE
added control on chevron icon & column for detailed table

### PR DESCRIPTION
**Pull request by omer2500**
i didn't know what to do with the selecRow() function so i made this solution 

please tell me what you think and if i need to change something

Thanks!

### CODE REVIEW

**Comment by omer2500** [comment]
i fixed what @wanxe  suggested.
about the third thing he mentioned
im not sure because of the selectRow() function.

**Comment by wanxe** [comment]
Thanks so much for the contribution, now we need to wait what says @jtommy :D

File: docs/pages/components/table/api/table.js [hunk 1]
```diff
@@ -170,6 +170,13 @@ export default [
                 values: '—',
                 default: '<code>false</code>'
             },
+            {
+                name: '<code>chevron</code>',
+                description: 'Allow chevron icon and column to be visible. if not will make the row click expand the row',
```
Comment (jtommy) [inline docs/pages/components/table/api/table.js]: Please change name to `showDetailIcon`

File: src/components/table/Table.vue [hunk 2]
```diff
@@ -66,13 +66,13 @@
                             :key="index"
                             :class="[rowClass(row, index), {
                                 'is-selected': row === selected,
-                                'is-checked': isRowChecked(row)
+                                'is-checked': isRowChecked(row),
                             }]"
-                            @click="selectRow(row)"
+                            @click="detailed && !chevron ? toggleOnclick(row) : selectRow(row)"
                             @dblclick="$emit('dblclick', row)">
```
Comment (jtommy) [inline src/components/table/Table.vue]: Remove it. I want to let free to toggle detail row using directly the method `toggleDetails(row)`, so a developer could be open the detail as he wants.

Comment (omer2500) [reply src/components/table/Table.vue]: i'm not sure i 100% understand you.
soon as i restore it,
the `toggleDetails(row)` wont be called if you hide the chevron column & icon, so the table will be just a normal table.

File: docs/pages/components/table/api/table.js (hunk 1 above)
Comment (omer2500) [reply docs/pages/components/table/api/table.js]: will fix! thanks

File: src/components/table/Table.vue (hunk 2 above)
Comment (jtommy) [reply src/components/table/Table.vue]: Yes a normal table and the developer choose how toggle detail section. For example click all row or button or custom icon

Comment (omer2500) [reply src/components/table/Table.vue]: got it.
i will make the changes and commit soon

**Comment by omer2500** [comment]
i fixed the issues based on your comments!

**Comment by jtommy** [comment]
@omer2500 Thanks!!!

**Pull request merged**